src/index.html

```diff
@@ -5,8 +5,299 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&amp;display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+               Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="main-catalog">
+        <div class="card main-catalog__card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Apple A1419"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <h3 class="card__code">Product code: 195434</h3>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__count">Reviews: 5</div>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price">$2,199</div>
+          </div>
+          <a
+            class="card__buy"
+            href="#"
+            data-qa="card-hover"
+          >
+          Buy
+          </a>
+        </div>
+        <div class="card main-catalog__card">
+          <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="Apple A1419"
+          >
```

Comment:
Still not fixed . Check other places
```suggestion
          <img
            class="card__image"
            src="images/imac.jpeg"
            alt="Apple A1419"
          >
```